sensors/omron_d6t/omron_d6t.py

```diff
@@ -83,13 +90,13 @@ def __parse_temperature_data(self, raw_sensor_buffer):
 
         return room_temperature, temperature
 
-    def __compute_crc(self, buf):
-        crc8_func = crcmod.mkCrcFun('crc-8')
+    def __calculate_pec(self, buf):
+        crc8_func = crcmod.predefined.mkCrcFun('crc-8')
 
-        # See OMRON datasheet for PEC details
-        pec_buffer = [self.__PEC_PREFIX]
+        pec_buffer = []
 
-        for itm in buf:
-            pec_buffer.append(itm)
+        # See OMRON datasheet for PEC details
+        pec_buffer.extend(self.__PEC_PREFIX)
+        pec_buffer.extend(buf)
```

Comment:
```python
pec_buffer.extend([self.__PEC_PREFIX, buf])
```

Reply:
This appends each array as an item in pec_buffer. So you would get something like this

pec_buffer = [ self.__PEC_PREFIX, buf ]
pec_buffer = [[...], [...]] 

Instead of actually appending the individual items of PEC_PREFIX and buf into pec_buffer

Reply:
Oh. Any reason on why you need actual arrays as items?

Reply:
Sorry! What I meant was the comment you supplied is what leads to having the arrays as items (which is not what we want!)

Reply:
Are you sure? What you said is true for `append` but not for extend from what I read 😕 
![image](https://user-images.githubusercontent.com/9994372/36236057-0de8a88a-11c2-11e8-806d-1aafad81f354.png)

Reply:
Maybe this clears it up a bit: 

![screen shot 2018-02-14 at 8 10 12 pm](https://user-images.githubusercontent.com/10552190/36236253-1f367ada-11c3-11e8-80c1-ed63f72eee26.png)

Reply:
Oh it's because `_PEC_PREFIX` and `buf` are lists as well.

Works for me then 👍